### DISCUSSION TITLE
[Request] Merge Playlists into a new Playlist

### DISCUSSION THREAD

**Discussion by BobWs** [navidrome/navidrome#3061] [Ideas]
Hi,
Would you please consider to add the ability to merge playlists into a new playlist, and if possible automatically shuffle the songs in the newly created playlist.

**Comment by BobWs** [comment]
For now I found a solution in Feishin. But if I may make a suggestion, why not team up with the developer of Feishin and implement his frontend to Navidrome. His playlist management works pretty well. There is always room for improvement...

**Comment by yllekz** [comment]
I was just thinking of this the other day. I noticed a lot of menu button/UI inconsistencies depending on where I was - for example, if I'm in a playlist, I can't select songs and add them to another playlist (which really needs to be a global ability)
